mitodl/open-discussions
# Add all other playlists option to config

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/2472

#### What's this PR do?
We will add a` id: wildcard option` to playlists to make it possible to download and make user lists for all playlists in a channel without listing each playlist in the config report individually. We will also add a ignore flag to ignore a playlist

For example:
```
channel_id: some_channel
playlists:
  - id: playlist1
    create_user_list: false
 - id: playlist2
    ignore: true
  -id: wildcard
   create_user_list: true
```
Will add all videos in the playlists in the channel except playlist2 to search and create user lists for every playlist except playlist1 and playlist2
#### How should this be manually tested?
Set
```
OPEN_VIDEO_DATA_BRANCH=format-change-2472
```
in your .env file.

Run 
```
from course_catalog.etl import pipelines
pipelines.youtube_etl(channel_ids="UCBpxspUNl1Th33XbugiHJzw")
```

This should add all the csail videos to search and create a UserList for every csail playlist except PLcwbNON2hqP5TFbkw0igt6Wz79WTaeFGi (Computer Vision)

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
